docs/_pages/web_client.md

```diff
@@ -186,14 +184,14 @@ getAllChannels()
 
 ### Calling methods on behalf of users
 
-When using [workspace tokens](https://api.slack.com/docs/working-with-workspace-tokens), some methods allow your app
-to perform the action [on behalf of a user](https://api.slack.com/docs/working-for-users). To use one of these methods,
+When using [User tokens/Personal tokens](https://api.slack.com/docs/token-types#user), some methods allow your app
```

Comment:
For consistency, we should invent the new term "Personal tokens". Let's just stick with "user tokens".